celi_framework/main.py

```diff
@@ -37,6 +39,34 @@
 
 logger = logging.getLogger(__name__)
 
+def get_default_config_path():
+    # Using importlib.resources to safely access the package resource
+    try:
+        config_path = resources.files('celi_framework.examples.wikipedia').joinpath('example_config.json')
+        # Temporary use during development:
+        # config_path = "/Users/jwag/PycharmProjects/celi/celi_framework/examples/wikipedia/example_config.json"
+        return str(config_path)
+    except Exception as e:
+        print(f"Failed to load default config: {e}")
+        return None
```

Comment:
IO think it's better to error out in this case then silently fail and run into problems later on.